### PR TITLE
MOSIP-18244: Cover the encryptor util tests

### PR DESCRIPTION
**Pull request by abuabdul**

### CODE REVIEW

File: resident/resident-service/src/test/java/io/mosip/resident/test/util/EncryptorUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        String encryptedJsonString = "{ data: ew0KCSJpZGVudGl0eSI6IHsNCgkJImRhdGVPZkJpcnRoIjogIjE5NzgvMDEvMDEiLA0KCQkiSURTY2hlbWFWZXJzaW9uIjogMC4xLA0KCQkiVUlOIjogIjkzNDIzNjEwMzgiDQoJfQ0KfQ }";
+
+        when(env.getProperty(Mockito.anyString())).thenReturn(encryptAPIUrl);
+        when(restClientService.postApi(Mockito.anyString(), Mockito.any(), Mockito.any(), Mockito.any(Class.class))).thenReturn(responseWrapper);
+        when(objectMapper.readValue(Mockito.anyString(), Mockito.any(Class.class))).thenReturn(decryptResponseDto);
+        when(objectMapper.writeValueAsString(responseWrapper.getResponse())).thenReturn(encryptedJsonString);
+
+        byte[] bytes = "Encrypt String".getBytes();
+        String refId = "CenterID_MachineID";
+        encryptorUtil.encrypt(bytes, refId);
```
Comment (vishwa-vyom) [inline resident/resident-service/src/test/java/io/mosip/resident/test/util/EncryptorUtilTest.java:89]: this method seems to be returning the entrypted string, can you do a assert on the return value ?

File: resident/resident-service/src/test/java/io/mosip/resident/test/util/EncryptorUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        when(DateUtils.getUTCCurrentDateTime()).thenReturn(localDateTime);
+    }
+
+    @Test
+    public void encryptTest() throws Exception {
+        DecryptResponseDto decryptResponseDto = new DecryptResponseDto();
+        decryptResponseDto.setData("ew0KCSJpZGVudGl0eSI6IHsNCgkJImRhdGVPZkJpcnRoIjogIjE5NzgvMDEvMDEiLA0KCQkiSURTY2hlbWFWZXJzaW9uIjogMC4xLA0KCQkiVUlOIjogIjkzNDIzNjEwMzgiDQoJfQ0KfQ");
+        io.mosip.resident.dto.ResponseWrapper<DecryptResponseDto> responseWrapper = new io.mosip.resident.dto.ResponseWrapper<>();
+        responseWrapper.setResponse(decryptResponseDto);
+        String encryptedJsonString = "{ data: ew0KCSJpZGVudGl0eSI6IHsNCgkJImRhdGVPZkJpcnRoIjogIjE5NzgvMDEvMDEiLA0KCQkiSURTY2hlbWFWZXJzaW9uIjogMC4xLA0KCQkiVUlOIjogIjkzNDIzNjEwMzgiDQoJfQ0KfQ }";
```
Comment (vishwa-vyom) [inline resident/resident-service/src/test/java/io/mosip/resident/test/util/EncryptorUtilTest.java]: If this is just a sample for encrypted data, remove "{ data:" and have only the encrypted string with any structure

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

**Review by m1054352 — APPROVED** [review]

**Review by mandeepdhiman123 — APPROVED** [review]

**Review by mandeepdhiman123 — APPROVED** [review]

**Review by mandeepdhiman123 — APPROVED** [review]

File: resident/resident-service/src/test/java/io/mosip/resident/test/util/EncryptorUtilTest.java (hunk 1 above)
Comment (abuabdul) [reply resident/resident-service/src/test/java/io/mosip/resident/test/util/EncryptorUtilTest.java:89]: As discussed, i am not asserting the returned value.

File: resident/resident-service/src/test/java/io/mosip/resident/test/util/EncryptorUtilTest.java (hunk 2 above)
Comment (abuabdul) [reply resident/resident-service/src/test/java/io/mosip/resident/test/util/EncryptorUtilTest.java]: I have updated this json string format.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**